instrumentation/jms/jms-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jms/JmsInstrumenterFactory.java

```diff
@@ -60,17 +61,19 @@ public Instrumenter<MessageWithDestination, Void> createConsumerReceiveInstrumen
     MessageOperation operation = MessageOperation.RECEIVE;
 
     // MessageConsumer does not do context propagation
```

Comment:
Is this comment outdated now?

Reply:
I removed the comment. I'm not quite sure what it means. It might be hinting that this particular instrumenter is used with `InstrumenterUtil.startAndEnd` where using `buildConsumerInstrumenter(MessagePropertyGetter.INSTANCE)` does not do context propagation and behaves like `builder.buildInstrumenter(SpanKindExtractor.alwaysConsumer())`